### DISCUSSION TITLE
Arctic AIO support?

### DISCUSSION THREAD

**Discussion by JoseHidalgo** [Rem0o/FanControl.Releases#1964] [General]
Hi everybody, I haven't found any info on this ("arctic" search on similar topics).
I just wonder: **does Fan Control support the current Arctic Cooling range of AIOs?**

I'm leaning towards an Arctic Freezer II 280 A-RGB, but first I'd like to make sure that it's well supported by Fan Control.

I was hoping to find a specific Fan Control plugin for this manufacturer, but I haven't found any. I'm a bit surprised since these AIOs are very popular and among the best in their class. Hopefully I'm just missing something here.

Could somebody enlighten me? Many thanks in advance! :)

![arctic_liquid_freezer_ii_280_argb_1](https://github.com/Rem0o/FanControl.Releases/assets/10851603/828fe8c0-798e-4c1d-a4d8-106441c78821)

**Comment by Antonimoo** [comment]
Hello there,
im running the Arctic Freezer II 360 RGB (non addressable) on a X570 AORUS ELITE (rev. 1.0) without any issues.
If you go to the support page of arctic right [here](https://support.arctic.de/pwm-settings) you can try to replicate all of the curves and choose which one you want to keep.
Best of luck!

**Reply by JoseHidalgo** [reply]
Thank you @Antonimoo ! A couple questions then:
- Is the pump detected, and can you set a curve to regulate the pump speed?
- Is the VRM fan detected?

**Reply by Antonimoo** [reply]
In short, the AIO appears like a single PWM fan.
Out of the box you have only one 4-pin PWM connector which controls the pump, VRM fan and radiator fans speed. There is a way to disconnect the fans connector from the main one, but im not sure its worth the effort to separate the control over the pump/VRM fan and radiator fans, just makes it more complicated imo.
Gamers nexus did a teardown and a good idepth review of the coolers, right [here](https://www.youtube.com/watch?v=ZQRwn-lecnk).
Cheers and go get that cooler, its awesome :D
